flask_humanize/__init__.py

```diff
@@ -89,6 +90,12 @@ def init_app(self, app):
             app.extensions = {}
         app.extensions['humanize'] = self
 
+        if app.config.get('HUMANIZE_USE_UTC'):
+            # Override humanize.time._now to use UTC time
+            humanize.time._now = datetime.utcnow
+        else:
+            humanize.time._now = datetime.now
+
```

Comment:
Patch `humanize.time._now` before registering extension in application, e.g. prepare environment and only then register extension themself.